app/app_main.c

```diff
@@ -130,12 +133,19 @@ char *ca_chain_file;
 char *cert_file;
 char *key_file;
 char *path_segment;
-static EVP_CIPHER_CTX cipher_ctx;  /* need to maintain across calls for MCT */
+
+static EVP_CIPHER_CTX *glb_cipher_ctx = NULL; /* need to maintain across calls for MCT */
 
 #define CHECK_ENABLE_CAP_RV(rv) \
     if (rv != ACVP_SUCCESS) { \
         printf("Failed to register capability with libacvp (rv=%d: %s)\n", rv, acvp_lookup_error_string(rv)); \
-        exit(1); \
+        return 1; \
+    }
+
+#define CHECK_ENABLE_CAP_RV_DEMO(rv) \
```

Comment:
Maybe I am missing it, where is this one used?

Reply:
Oops, this macro isn't needed here. It was intended for mirror. I'll remove